stdlib/ballerina-builtin/src/main/ballerina/auth/Package.md

```diff
@@ -12,4 +12,18 @@ When creating a new authentication provider, there are two functions that need t
 - `authenticate` : authenticate the user based on a credential, which can be username/password or a token such as JWT.
 - `getScopes` : provide the scopes associated with the user. Scopes are primarily permissions that are required to access a protected resource. 
 
+### Config Auth Provider
+
+Config Auth Provider is an implementation of the Auth Provider interface, which uses the ballerina.conf file to store
```

Comment:
Config Auth Provider should  be formatted as code, using back ticks -> `ConfigAuthProvider`
Same goes for Auth Provider -> `AuthProvider` and ballerina.conf -> `ballerina.conf`